.buildkite/x-pack/pipeline.xpack.filebeat.yml

```diff
@@ -97,9 +92,9 @@ steps:
           - "x-pack/filebeat/build/*.json"
         notify:
           - github_commit_status:
-              context: "x-pack/filebeat: Windows 2022 Unit Tests"
+              context: "x-pack/filebeat: Win-2022 Unit Tests"
 
-      - label: ":windows: Windows 2016 Unit Tests"
+      - label: ":windows: Win-2016 Unit Tests"
```

Comment:
why we abbreviate, having explicit the name Windows seems ok

Reply:
In some previous beat PRs it was abbreviated as well, so I just followed that pattern.